lms/djangoapps/verify_student/models.py

```diff
@@ -463,23 +452,18 @@ def approve(self, user_id=None, service=""):
         )
         self.save()
 
-        # Emit event to find and generate eligible certificates
-        verification_data = VerificationAttemptData(
-            attempt_id=self.id,
-            user=UserData(
-                pii=None,
-                id=self.user.id,
-                is_active=self.user.is_active,
-            ),
-            status=self.status,
-            name=self.name,
-            expiration_date=self.expiration_datetime,
-        )
-        IDV_ATTEMPT_APPROVED.send_event(
-            idv_attempt=verification_data,
+        # Emit signal to find and generate eligible certificates
+        # This model uses the LEARNER_SSO_VERIFIED signal for backwards compatibility with changes
+        # to existing records. This model is being deprecated and all future IDV records
+        # will use the new VerificationAttempt model and corresponding openedx events signal.
+        # This is temporary until this model can be removed.
+        # DEPR details: https://github.com/openedx/edx-platform/issues/35128
+        LEARNER_SSO_VERIFIED.send_robust(
```

Comment:
I read through your explanation of this choice, but I'm still a little unsure of why this change is in the `PhotoVerification` model and not the `SSOVerification` model. I'll try explaining it in the way that makes sense to me - could you let me know if I have this right?

`PhotoVerification` is the ABC of the `SoftwareSecurePhotoVerification`. `SoftwareSecurePhotoVerification` approvals should continue to be caught by the `certificates` application. Because the `approve` method is on the `PhotoVerification` model, that's where we emit this event.

Reply:
If that's the case, I think `LEARNER_SSO_VERIFIED` is a confusing name. I'm kind of torn on it, especially because who knows when this model will be deprecated and removed. If the intention is to use the name that will make the most sense in the future when `PhotoVerification` is deprecated, I think a brief ADR codifying this choice would be helpful. Alternatively, we could use a more generic name and leave the renaming from the generic name to `LEARNER_SSO_VERIFIED` to whomever refactors this codebase during deprecation. What do you think?

Reply:
yeah that's correct, it's where this event has always been emitted. 

I'm kinda stumped on the name because if I call it something generic (like learner_verified) is also becomes wrong in the sense that it doesn't actually capture all cases of learner_verified just this one off instance. I'll think on this more

Reply:
okay after mulling on this, I can just create a separate event here for PhotoVerification that uses the same cert handler. Less confusion, and the 2nd can just be deleted up during the DEPR. @MichaelRoytman thoughts?

Reply:
I think that's a good plan!